### DISCUSSION TITLE
"NaN" in my report using Newman

### DISCUSSION THREAD

**Discussion by marianagroppa** [test-results-reporter/reporter#89] [Q&A] [answered]
I'm using Newman for my tests.
After running it gives me an XML report.

`newman run MyCollection.json --insecure -r junit -n 1`

Report in XML:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="MyCollection" tests="1" time="32.937">
  <testsuite name="1 - Authenticate " id="u68c4966-0877-4766-1e5b-uu96efb7f84" timestamp="2022-09-20T16:49:56.147Z" tests="1" failures="0" errors="0" time="0.807">
    <testcase name="Teste" time="0.807" classname="TestePostamn"/>
  </testsuite>
 </testsuites> 
 ```

When I send it to Teams, the report doesn't display correctly.
Displays as "NaN"
![teams](https://user-images.githubusercontent.com/35785141/191537420-e2446adb-448a-4292-a9b6-48c6f29ebc3c.png)

My config.json:
```json
{
  "reports": [
    {
      "targets": [
        {
          "name": "teams",
          "condition": "passOrFail",
          "inputs": {
            "url": "www.urlTeams.com",
            "publish": "test-summary-slim",
            "title": "Teste Result Postman"
          },
          "extensions": [
            { "name": "quick-chart-test-summary" },
            {
              "name": "hyperlinks",
              "inputs": {
                "links": [
                  {
                    "text": "Build",
                    "url": "www.urlBuild.com"
                  },
                  {
                    "text": "Report",
                    "url": "www.urlReport.com"
                  }
                ]
              }
            },
            {
              "name": "mentions",
              "inputs": {
                "users": [
                  {
                    "name": "Mariana Groppa",
                    "teams_upn": "mariana@email.com.br"
                  }
                ]
              }
            }
          ]
        }
      ],
      "results": [
        {
          "type": "junit",
          "files": ["newman-run-report-full-2022-09-20-18-02-10-539-0.xml"]
        }
      ]
    }
  ]
}
```

What could I be doing wrong?
Using other testing frameworks, it works well! (Playrwright, Pactum)

**Comment by ASaiAnudeep** [comment, marked as answer]
Hello @marianagroppa , 

Usually the junit xml file includes a `failures` attribute in `testsuites` tag. Please find the example [here](https://github.com/test-results-reporter/parser/blob/main/tests/data/junit/single-suite.xml)

Need to investigate further. 

> 08/10/22

@marianagroppa @sgolyala1 - fixed it in the latest version [v1.0.11](https://github.com/test-results-reporter/reporter/releases/tag/v1.0.11)

**Reply by sgolyala1** [reply]
Hi @ASaiAnudeep  - I now see the values instead of NaN - but the values(using for calculation) are not correct 
![image](https://user-images.githubusercontent.com/78226609/194725522-ccbf0560-629c-47d3-ae6a-0e70ddbce7f4.png)

**Reply by sgolyala1** [reply]
[junitReport.txt](https://github.com/test-results-reporter/reporter/files/9740208/junitReport.txt)
This is my test report and I have only 4failures out of 22test cases so Result to be 18/22(81.81%)

**Reply by ASaiAnudeep** [reply]
@sgolyala1 - fixed it in the latest version [v1.0.13](https://github.com/test-results-reporter/reporter/releases/tag/v1.0.13)

**Reply by sgolyala1** [reply]
@ASaiAnudeep Looks great(with all passed atm)!!Thankyou for sorting this out so quickly 
![image](https://user-images.githubusercontent.com/78226609/194752786-7f22d431-684a-47d6-adc9-014a6147705f.png)
🙏👌
NB:Will get back to you again if I need anything else(Didn't check when there are failures)

**Reply by marianagroppa** [reply]
Tksss, @ASaiAnudeep !!!
You are the best! :)

**Reply by ASaiAnudeep** [reply]
@sgolyala1 thanks for reporting this issue. Can you please start an independent discussions for both the above issues.

**Reply by sgolyala1** [reply]
sure!!Raised as New

**Comment by sgolyala1** [comment]
hi I'm having the same issue as Mariana mentioned - seeing NaN - Please try to look into this .Thankyou
![image](https://user-images.githubusercontent.com/78226609/194594669-843ca5cd-60df-450f-a798-bd1784847003.png)
